website/docs/r/datasource_google_iam_role.html.markdown

```diff
@@ -0,0 +1,37 @@
+---
+layout: "google"
+page_title: "Google: google_iam_role"
+sidebar_current: "docs-google-datasource-folder"
```

Comment:
This should be `docs-google-datasource-iam-role`

Reply:
You'll also need to add this datasource to `google.erb` so it shows in the sidebar.

Reply:
also just noticed that I put originally under `/r` and not under `/d` where its suppose to be. fixed now

Reply:
Nice catch!